assets/src/edit-story/components/library/panes/media/media3p/media3pCategories.js

```diff
@@ -18,34 +18,78 @@
  * External dependencies
  */
 import styled from 'styled-components';
+import { useState } from 'react';
 
 /**
  * Internal dependencies
  */
+import { ArrowDownButton } from '../../../../button/index';
 import CategoryPill from './categoryPill';
-
 // TODO(#2360) Category should be collapsible and expandable.
 const CategorySection = styled.div`
   background-color: ${({ theme }) => theme.colors.bg.v3};
   min-height: 94px;
-  padding: 30px 24px;
+  padding: 30px 24px 10px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+`;
+
+// This hides the category pills unless expanded
+const CategoryPillContainer = styled.div`
+  height: ${(props) => (props.isExpanded ? 'auto' : '36px')};
   display: flex;
   flex-wrap: wrap;
+  overflow: hidden;
 `;
 
+// Flips the button upside down when expanded;
+const ExpandButton = styled(ArrowDownButton)`
+  ${(props) => props.isExpanded && 'transform: matrix(1, 0, 0, -1, 0, 0);'};
+`;
 // TODO(#2362) Wire up pill list to state and remove these fake pills.
-const fakePillList = ['COVID-19', 'Nature', 'Wallpapers', 'People'];
-const selectedPill = 'Nature';
+const fakePillList = [
+  'COVID-19',
+  'Nature',
+  'Wallpapers',
+  'People',
+  'Texture & Pattern',
+  'Business & Work',
+  'Travel',
+  'Technology',
+  'Animals',
+  'Interiors',
+  'Architecture',
+  'Food & Drinks',
+];
+
+const Media3pCategories = () => {
+  const [isExpanded, setIsExpanded] = useState(false);
+  const [selectedPill, setSelectedPill] = useState(-1);
 
-const Media3pCategories = () => (
-  <CategorySection>
-    {
-      // TODO(#2362) Wire up pill list to state and remove these fake pills.
-      fakePillList.map((e, i) => (
-        <CategoryPill isSelected={e == selectedPill} key={i} title={e} />
-      ))
-    }
-  </CategorySection>
-);
+  return (
+    <CategorySection>
+      <CategoryPillContainer isExpanded={isExpanded}>
+        {
+          // TODO(#2362) Wire up pill list to state and remove these fake pills.
+          fakePillList.map((e, i) => (
+            <CategoryPill
+              isSelected={i == selectedPill}
```

Comment:
Can multiple pills be selected? If so, `aria-pressed` might be suitable here.

If not, `CategoryPillContainer` should have `role="tablist"`, and the pills should have `role="tab"`. See https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/Tab_Role

Let's not make accessibility be an after thought 🙂

Reply:
1) I added role="tablist" to the CategoryPillContainer. 

2) I added role="tab" and aria-selected={props.isSelected} to the CategoryPill component in categoryPill.js.